# 8thICMBB2025
## Poster Presentation: PE007
##  Methanogenic Archaeal Class Bog-38 In the North Selangor Peat Swamp Forest: A Tropical Outlier In a Predominantly Arctic Lineage
More details on my poster presentation


## References

1. Johannes Alneberg, Brynjar Smári Bjarnason, Ino de Bruijn, Melanie Schirmer, Joshua Quick, Umer Z Ijaz, Leo Lahti, Nicholas J Loman, Anders F Andersson, and Christopher Quince. Binning metagenomic contigs by coverage and composition. Nature Methods, 11(11):1144–1146, September 2014. ISSN 1548-7105. doi: 10.1038/nmeth.3103. URL http://dx.doi.org/10.1038/nmeth.3103.
2. Global Environment Centre. Integrated Management Plan for North Selangor Peat Swamp Forest 2014-2023. Selangor State Forestry Department, 2014. ISBN 9789671026854.
3. Pierre-Alain Chaumeil, Aaron J Mussig, Philip Hugenholtz, and Donovan H Parks. Gtdb-tk v2: memory friendly classification with the genome taxonomy database. Bioinformatics, 38(23):5315–5316, October 2022. ISSN 1367-4811. doi: 10.1093/bioinformatics/btac672. URL http://dx.doi.org/10.1093/ bioinformatics/btac672.
4. Mikhail Kolmogorov, Jeffrey Yuan, Yu Lin, and Pavel A. Pevzner. Assembly of long, error-prone reads using repeat graphs. Nature Biotechnology, 37(5): 540–546, April 2019. ISSN 1546-1696. doi: 10.1038/s41587-019-0072-8. URL http://dx.doi.org/10.1038/s41587-019-0072-8.
5. Michael D Lee. Gtotree: a user-friendly workflow for phylogenomics. Bioinformatics, 35(20):4162–4164, March 2019. ISSN 1367-4811. doi: 10.1093/bioinformatics/btz188. URL http://dx.doi.org/10.1093/bioinformatics/btz188.
6. Vijini Mallawaarachchi and Yu Lin. Accurate binning of metagenomic contigs using composition, coverage, and assembly graphs. Journal of Computational Biology, 29(12):1357–1376, December 2022. ISSN 1557-8666. doi: 10.1089/cmb.2022.0262. URL http://dx.doi.org/10.1089/cmb.2022.0262.
7. Rhiannon Mondav, Ben J. Woodcroft, Eun-Hae Kim, Carmody K. McCalley, Suzanne B. Hodgkins, Patrick M. Crill, Jeffrey Chanton, Gregory B. Hurst, Nathan C. VerBerkmoes, Scott R. Saleska, Philip Hugenholtz, Virginia I. Rich, and Gene W. Tyson. Discovery of a novel methanogen prevalent in thawing permafrost. Nature Communications, 5(1), February 2014. ISSN 2041-1723. doi: 10.1038/ncomms4212. URL http://dx.doi.org/10.1038/ncomms4212.
8. Shaojun Pan, Chengkai Zhu, Xing-Ming Zhao, and Luis Pedro Coelho. A deep siamese neural network improves metagenome-assembled genomes in microbiome datasets across different environments. Nature Communications, 13(1), April 2022. ISSN 2041-1723. doi: 10.1038/s41467-022-29843-y. URL http://dx.doi.org/10.1038/s41467-022-29843-y.
9. Michael Shaffer, Mikayla A Borton, Bridget B McGivern, Ahmed A Zayed, Sabina Leanti La Rosa, Lindsey M Solden, Pengfei Liu, Adrienne B Narrowe, Josué Rodríguez-Ramos, Benjamin Bolduc, M Consuelo Gazitúa, Rebecca A Daly, Garrett J Smith, Dean R Vik, Phil B Pope, Matthew B Sullivan, Simon Roux, and Kelly C Wrighton. Dram for distilling microbial metabolism to automate the curation of microbiome function. Nucleic Acids Research, 48(16):8883–8900, August 2020. ISSN 1362-4962. doi: 10.1093/nar/gkaa621. URL http://dx.doi.org/10.1093/nar/gkaa621.
10. Ben J. Woodcroft, Caitlin M. Singleton, Joel A. Boyd, Paul N. Evans, Joanne B. Emerson, Ahmed A. F. Zayed, Robert D. Hoelzle, Timothy O. Lamberton, Carmody K. McCalley, Suzanne B. Hodgkins, Rachel M. Wilson, Samuel O. Purvine, Carrie D. Nicora, Changsheng Li, Steve Frolking, Jeffrey P. Chanton, Patrick M. Crill, Scott R. Saleska, Virginia I. Rich, and Gene W. Tyson. Genome-centric view of carbon processing in thawing permafrost. Nature, 560(7716):49–54, July 2018. ISSN 1476-4687. doi: 10.1038/s41586-018-0338-1. URL http://dx.doi.org/10.1038/s41586-018-0338-1.